docs/ml/deploy/tflite_cpu.md

```diff
@@ -0,0 +1,133 @@
+- a model available on your machine
+
+To configure a `tflite_cpu` ML model service:
+
+{{< tabs >}}
+{{% tab name="Builder" %}}
+
+Navigate to your machine's **Config** tab on the [Viam app](https://app.viam.com/robots).
+Click the **Services** subtab and click **Create service** in the lower-left corner.
+Select the `ML Model` type, then select the `TFLite CPU` model.
+Enter a name for your service and click **Create**.
+
+You can choose to configure your service with an existing model on the machine or deploy a model onto your machine:
+
+{{< tabs >}}
+{{% tab name="Deploy Model on Robot" %}}
+
+1. To configure your service and deploy a model onto your machine, select **Deploy Model On Robot** for the **Deployment** field.
+
+2. Click on **Models** to open a dropdown with all of the ML models available to you privately, as well as all of the ML models available in [the registry](https://app.viam.com), which are shared by users.
+   You can select from any of these models to deploy on your robot.
+
+{{<imgproc src="/services/deploy-model-menu.png" resize="700x" alt="Models dropdown menu with models from the registry.">}}
+
+{{% alert title="Tip" color="tip" %}}
+To see more details about a model, open its page in [the registry](https://app.viam.com).
+{{% /alert %}}
+
+3. Also, optionally select the **Number of threads**.
+
+{{<imgproc src="/services/deploy-model.png" resize="700x" alt="Create a machine learning models service with a model to be deployed">}}
+
+{{% /tab %}}
+{{% tab name="Path to Existing Model On Robot" %}}
+
+1. To configure your service with an existing model on the machine, select **Path to Existing Model On Robot** for the **Deployment** field.
+2. Then specify the absolute **Model Path** and any **Optional Settings** such as the absolute **Label Path** and the **Number of threads**.
+
+![Create a machine learning models service with an existing model](/services/available-models.png)
+
+{{% /tab %}}
+{{< /tabs >}}
+
+{{% /tab %}}
+{{% tab name="JSON Template" %}}
+
+Add the `tflite_cpu` ML model object to the services array in your raw JSON configuration:
+
+```json {class="line-numbers linkable-line-numbers"}
+"services": [
+  {
+    "name": "<mlmodel_name>",
+    "type": "mlmodel",
+    "model": "tflite_cpu",
+    "attributes": {
+      "model_path": "${packages.ml_model.<model_name>}/<model-name>.tflite",
+      "label_path": "${packages.ml_model.<model_name>}/labels.txt",
+      "num_threads": <number>
+    }
+  },
+  ... // Other services
+]
+```
+
+{{% /tab %}}
+{{% tab name="JSON Example" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+"services": [
+  {
+    "name": "fruit_classifier",
+    "type": "mlmodel",
+    "model": "tflite_cpu",
+    "attributes": {
+      "model_path": "${packages.ml_model.my_fruit_model}/my_fruit_model.tflite",
+      "label_path": "${packages.ml_model.my_fruit_model}/labels.txt",
+      "num_threads": 1
+    }
+  }
+]
```

Comment:
I think here we definitely want to add packages in the config

Reply:
added